example-app/SantokuApp/src/features/qa-event/components/EventList.tsx

```diff
@@ -18,25 +20,33 @@ export type EventListProps = {
 
 export const EventList: React.FC<EventListProps> = ({data, containerWidth}) => {
   const theme = useTheme<RestyleTheme>();
+  const {width: windowWidth} = useSafeAreaFrame();
   const eventListCardWidth = useMemo(() => containerWidth - theme.spacing.p24 * 2, [containerWidth, theme.spacing.p24]);
```

Comment:
```suggestion
  const eventListCardWidth = useMemo(() => containerWidth - theme.spacing.p8 * 2, [containerWidth, theme.spacing.p8]);
```
余白計算は以下の部分と対応しているので、変更が必要です。

```diff
-          <StyledSpace width="p24" />
-          <EventListCard event={item} containerWidth={eventListCardWidth} />
-          <StyledSpace width="p24" />
+          <StyledSpace width="p8" />
+          <EventListCard event={item} containerWidth={eventListCardWidth} />
+          <StyledSpace width="p8" />
```

Reply:
添付した画像の赤枠部分がcontainerWidthから引く値かと思います。
中心に映っているカードの左余白(8px)+左に映っているカードの右余白(8px)+左に映っているカードの
はみ出し部分(8px) =24pxという計算になるのではないかと思いました。
![image](https://user-images.githubusercontent.com/38860388/227854273-319303df-9ef6-416a-a8c5-3dcbbda86fc5.png)

Reply:
なるほど、`eventListCardWidth` の値は変わらずに、余白部分(24)でやりくりしているので、 `p24` を変更するとずれるってことですね。

でしたら、 `p24` を使わずに表現しましょう。
コメントでも説明しておくと良いです。

あるいは、可能ならば、コードで表現するのもありです。
※ 以下の `width` 部分に数値計算結果を使えるかは未確認

- `<StyledSpace width="p16" />`
- `<StyledSpace width="p8" />`

Reply:
一旦、こちらで算出式の変更とコメントでの説明を追記しております。
https://github.com/ws-4020/mobile-app-crib-notes/pull/1082/commits/2921fac928a1a824ea9c8a287842d794fae4f5ab